hydra/test_utils/config_source_common_tests.py

```diff
@@ -13,11 +13,13 @@
 class ConfigSourceTestSuite:
     def skip_overlap_config_path_name(self) -> bool:
         """
-        Some config source plugins do not support config name and path overlap, for example:
+        Some config source plugins do not support config name and path overlap.
+        For example the following may not be allowed:
+        (dataset exists both as a config object and a config group)
         /dateset.yaml
         /dataset/cifar.yaml
-        may not be allowed.
-        """
+
+        Override this function to return True so the test assumes /dataset/cifar.yaml exists instead of /dataset.yaml"""
```

Comment:
what?
why does it mean cifar.yaml exists INSTEAD of dataset.yaml? those two things seems unrelated.

Reply:
in configerator's case that  is what we want to test to think right? (we cannot have dataset.yaml and /dataset/cifar.yaml both exist)

Reply:
so we exclude dataset.yaml (cconf) from the test, right?

Reply:
Yea,  that's the plan :) 
Are you suggesting a wording change here?

Reply:
yes. the comment there has nothing to do with how we are solving the problem as far as I can tell.

Reply:
Got it, will update the doc.